rpcs3/Emu/System.h

```diff
@@ -245,6 +257,17 @@ class Emulator final
 
 	std::set<std::string> GetGameDirs() const;
 
+	u64 GetEmulationCounter() const
```

Comment:
Is this even used anywhere?

Reply:
I can think of conditions where it will be used in the future, most likely by me.